src/views/components/LoginForm/component.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { Button, Col, Layout, Row, Typography, Form as AntForm } from 'antd'
+import { LockOutlined, UserOutlined } from '@ant-design/icons'
+import { Form, Field } from 'formik'
+
+import InputField from 'Views/components/shared/InputField'
+import { defaultProps, propTypes } from './types'
+
+const LoginFormComponent = ({ isSubmitting, status }) => {
```

Comment:
simply `LoginForm`

Reply:
Are you sure? We have container, maybe better container have name LoginForm?

Reply:
to add endings like `Component` or `Container` can be useful, but not in this case, because you don't have any name collisions here